resources/views/includes/css/css.blade.php

```diff
@@ -22,3 +22,5 @@
 @hasSection('360')
     <link rel="stylesheet" href="{{ asset('/css/pannellum.css') }}" />
 @endif
+
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@splidejs/splide@4.1.4/dist/css/splide.min.css" />
```

Comment:
Ideally we would self host CSS and JS files for security and performance reasons, although this is more important for JavaScript files (for security reasons).

There are other CSS files that are hosted by a third party, so it's not something that has to be changed to this pull request, but just to note for future work.